docs/superquickstart.md

```diff
@@ -1,42 +1,62 @@
 
 The Superquick start interface enables you to rapidly explore your data without having to create an account or install any software besides your web browser and data conversion programs.
 
-**IMPORTANT!**: The limitations to this method is that the files uploaded will not reside in your own account (though no one else can see them) and will removed on a monthly basis. Please create your own account for more persistent and reproducible analyses.
+## No log-in required
+
+**IMPORTANT!**: The limitations to this method is that the files uploaded will not reside in your own account (though no one else can see them) and will be removed on a monthly basis. Please create your own account for more persistent and reproducible analyses.
 
 You would be able to use the Super quickstart interface if you have the following requirements:
 
-- [ ] The format of your files is mzXML/mzML/Thermo raw files.
+- [ ] The format of your files is mzXML/mzML (Thermo raw files (.raw)/Bruker raw files (.d) can be converted on GNPS [Checkout this [documentation](fileconversion.md).])
```

Comment:
This is incorrect. Quickstart does not support Bruker files.

Reply:
Bruker files can't be directly used, but they can be converted with the fileconversion tool, right? I can remove from here. How do I make changes to the commit?

Reply:
I removed from the description in the new commit.